src/components/PlantingSites/BoundariesAndPlots.tsx

```diff
@@ -0,0 +1,61 @@
+import { Typography, Grid, Box, CircularProgress } from '@mui/material';
+import { Button, theme } from '@terraware/web-components';
+import strings from 'src/strings';
+import { TERRAWARE_SUPPORT_LINK } from 'src/constants';
+import { PlantingSite } from 'src/api/types/tracking';
+import { PlantingSiteMap } from '../Map';
+
+type BondariesAndPlotsProps = {
```

Comment:
typo, should be BoundariesAndPlotsProps?

Reply:
Thanks!